src/main/resources/templates/languages/cy/non-strategic/rptEasternWeeklyHearingList.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I am not sure if repeating this file here 5 times. Essentially they are the same file for each region. For example, SSCS_DAILY_LIST_ADDITIONAL_HEARINGS have got a field called parentListType in the ListType enum and it is able to reuse the resource from the parent type. May be we should find a way for them to use a common file

Reply:
as discussed, it's done.